Quick Start Docs Callout (#875)
tinacms/tinacms.org: content/docs/index.md

- The **TinaCMS toolkit**, a suite of NPM packages that enable developers to quickly create a custom content management frontend for a website.
- **Tina Cloud**, a hosted service designed to make Git-based content management more powerful and accessible for cross-functional teams.

<div class="callout">
<img className="learnImage" src="../img/tina-laptop.png" alt="" />
  <div>
  <h3>Ready to get started?</h3>

>   <p>Read our getting started guide to get set up in no time.</P>

Reviewer: ```suggestion
  <p>Get a Tina running website in no time!</p>
```

Author: @DirtyF  How about "Get a website running with Tina Cloud in no time!"?

Reviewer: even better!

Author: Done.
